### PR TITLE
Upgrading Teams JS SDK version to beta version 1.11.0-beta.2

### PR DESCRIPTION
**Pull request by samyuktha24**
Upgrading Teams JS SDK version to beta version 1.11.0-beta.2. As part of the upgrade, fixed build issue related to registerChangeSettingsHandler as the function has been renamed to registerEnterSettingsHandler in Teams JS SDK

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -85,11 +85,11 @@ export const initializeAppModules = () => {
 
 
     addModule({
-      name: "registerChangeSettingsHandler",
+      name: "registerEnterSettingsHandler",
```
Comment (sachingupta) [inline src/app.ts:88]: why this change.

**Review by sachingupta — APPROVED** [review]

File: src/app.ts (hunk 1 above)
Comment (samyuktha24) [reply src/app.ts:88]: registerChangeSettingsHandler method has been renamed to registerEnterSettingsHandler in Teams JS SDK. On upgrading to 1.11.0-beta.2, the Yuri Test tab app build was failing as registerChangeSettingsHandler is no longer available in latest JS SDK. Its renamed to registerEnterSettingsHandler.
